youtube_dl/extractor/bbc.py

```diff
@@ -797,7 +797,8 @@ def _real_extract(self, url):
                                 playlist.get('progressiveDownloadUrl'), playlist_id, timestamp))
 
         if entries:
-            playlist_title = playlist_title or remove_end(self._og_search_title(webpage), ' - BBC News')
+            playlist_title = playlist_title or remove_end(
+                self._og_search_title(webpage, default=' - BBC News'), ' - BBC News')
```

Comment:
There is a playlist title that can be extracted. Defaulting to ` - BBC News` in order to replace it then by ` - BBC News` looks clumsy.